config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -69,13 +25,13 @@ periodics:
         - "./scripts/ci-e2e.sh"
       env:
         - name: KUBERNETES_VERSION_UPGRADE_FROM
-          value: "stable-1.20"
-        - name: KUBERNETES_VERSION_UPGRADE_TO
           value: "stable-1.21"
+        - name: KUBERNETES_VERSION_UPGRADE_TO
+          value: "stable-1.22"
         - name: ETCD_VERSION_UPGRADE_TO
-          value: "3.4.13-0"
+          value: "3.5.1-0"
```

Comment:
We want to be careful about suggesting to users to use etcd 3.5, but we especially don't want to use 3.5.1. See:

https://groups.google.com/g/etcd-dev/c/sad9tgmKU7Y/m/vhArFVevBgAJ

Reply:
updated to 3.5.3